scripts/core/superdesk-api.d.ts

```diff
@@ -75,6 +75,7 @@ declare module 'superdesk-api' {
                     onSpikeMultiple?(items: Array<IArticle>): Promise<onSpikeMiddlewareResult>;
                     onPublish?(item: IArticle): Promise<onPublishMiddlewareResult>;
                     onRewriteAfter?(item: IArticle): Promise<IArticle>;
+                    onSendBefore?(itemIds: Array<string>, desk: IDesk): Promise<void>;
```

Comment:
Could you make it `items: Array<IArticle>` instead of IDs only? I think the items are already fetched at this point and it'd be faster to resolve the promise when extensions don't have to fetch items themselves.